Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -133,11 +138,17 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
     });
 
     return (
-      <View style={[ styles.appbar, style ]} {...viewProps}>
+      <Animated.View style={[
+          styles.appbar,
+          { height: APPBAR_HEIGHT + this.props.statusBarHeight },
```

Comment:
Btw, it is possible to add a plain number with an animated value, but it looks kinda funny:

```
height: Animated.add(animatedHeight, new Animated.Value(5)),
```